daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -13,8 +13,10 @@ import com.daml.lf.interpretation.{Error => IE}
 import com.daml.lf.language.Ast
 import com.daml.lf.speedy.SError._
 import com.daml.lf.speedy.SExpr._
-import com.daml.lf.speedy.Speedy._
 import com.daml.lf.speedy.SResult._
+import com.daml.lf.speedy.Speedy._
+import com.daml.lf.speedy.{SExpr0 => s}
+import com.daml.lf.speedy.{SExpr => t}
```

Comment:
Do not rename here,
-  convention source/target does not make sens here 
- the file should not use SExpr0 (We can tackle the second point is an separate PR)

Reply:
I stopped using the renames `source`/`target`.